cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsUpscaleService.java

```diff
@@ -104,6 +104,13 @@
 
         awsTaggingService.tagRootVolumes(ac, amazonEC2Client, instances, stack.getTags());
 
+        // TODO: Reduce the polling interval from 10 seconds to something like 3-5 seconds.
+        //  Alternately a dynamic polling interval, with 'expectedRunTime' as a parameter.
+        //  e.g. 50 seconds to launch an instance - would poll every 10-15 seconds until the 40s mark,
+        //  after which it switches to 2-3 seconds.
```

Comment:
if we poll too often we have a higher chance of hitting throttling errors

Reply:
Agreed. We're not talking orders of magnitude increase though.